sdks/python/apache_beam/io/datastore/v1/fake_datastore.py

```diff
@@ -48,6 +48,20 @@ def run_query(req):
   return run_query
 
 
+def create_commit(mutations):
+  """A fake datastore commit method that writes the mutations to a list.
+
+  Args:
+    mutations: A list to write mutations to.
+  """
+
```

Comment:
There is no need for empty space after docstring. Similarly one empty line is enough between functions.